OrbitQt/orbitmainwindow.cpp

```diff
@@ -596,6 +596,13 @@ void OrbitMainWindow::on_actionReport_Bug_triggered() {
   }
 }
 
+void OrbitMainWindow::on_actionOpen_User_Data_Directory_triggered() {
+  if (!QDesktopServices::openUrl(
+          QUrl(Path::CreateOrGetOrbitAppDataDir().string().c_str(), QUrl::StrictMode))) {
+    QMessageBox::critical(this, "Error opening URL", "Could not open Orbit directory");
```

Comment:
Nit: The "URL" in the title seems quite misleading to users, I'd replace it with "directory".